src/multi-selector.js

```diff
@@ -463,7 +463,7 @@ const MultiSelector = React.createClass({
 						<div
 							style={{ padding: '8px 16px', borderTop: '1px solid rgb(233, 233, 233)' }}
 							className={`cps-bg-gray-3 ${styles['cpr-multi-selector__done']}`}>
-							<a onClick={this.closeDialog.bind(this)} className={`cps-link +primary`}>
+							<a onClick={this.closeDialog} className={`cps-link +primary`}>
```

Comment:
Are we getting rid of this just for performance reasons? Or was there a bug related to this?

Reply:
This is to make happy the warning I was getting: `You are binding a component method to the component. React does this for you automatically in a high-performance way, so you can safely remove this call.`